media/css/externalpages/pocket/components/pocket-hits-signup.scss

```diff
@@ -0,0 +1,209 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '../includes/nav';
+@import '../includes/footer';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/base/elements/containers';
+
+.pocket-hits-header {
+    @include text-title-sm;
+    color: $color-text-primary;
+    font-family: $font-sans-alt;
+    font-size: 40px;
+    font-weight: 400;
+    margin: 0 0 $spacing-md;
+    text-align: center;
+}
+
+.pocket-hits-subheader {
+    @include text-body-xl;
```

Comment:
**nit:** remove this too since we already have `font-size` defined for this class